web/src/app/schedules/temp-sched/TempSchedShiftsList.tsx

```diff
@@ -57,33 +71,99 @@ export default function TempSchedShiftsList({
       isValid: schedInterval.engulfs(parseInterval(s)),
     }))
 
-    if (!shifts.length) return []
-
-    const lastShift = shifts.reduce(
-      (result, candidate) => (candidate.end > result.end ? candidate : result),
-      shifts[0],
-    )
-
-    const firstShiftStart = shifts[0].start
-    const lastShiftEnd = lastShift.end
-
     const displaySpan = Interval.fromDateTimes(
-      DateTime.min(schedInterval.start, firstShiftStart).startOf('day'),
-      DateTime.max(schedInterval.end, lastShiftEnd).endOf('day'),
+      DateTime.fromISO(start).startOf('day'),
+      DateTime.fromISO(end).endOf('day'),
     )
```

Comment:
As is I wasn't able to reproduce this. When I follow steps 1-3, an error message shows at the bottom of step 1 displays `One or more shifts extend beyond the start and/or end of this temporary schedule`. In which case they would go and delete the shift and then they could redo from step 1.

Reply:
Right now we display error messages but still allow clicking the button, and the button also gets stuck on retry if a request fails instead of back/next if needed. Definitely more follow-up work will be needed here

Reply:
What I can do while we work on that UX for the buttons is still render the shifts that fall outside of coverage without notice info. The logic for the notices will need to stay the same for reasons I mentioned above so we won't be able to use `max` there, but we can still render them past that, and the error message will be displayed below. It's been more than a few months since I've looked at this body of work, but I'll check out what it'll take for something like that.